Bump and release zenko 2.6.70

francoisferrand opened the pull request:
- Merge #2168
- Merge #2169
- Bump backbeat 8.6.51
- Bump zenko-operator 1.6.5
- Bump cloudserver 8.8.36
- Bump sorbet 1.1.13
- Release Zenko 2.6.70

Issue: ZENKO-4919

francoisferrand committed:
Add artesca-root-ca-issuer cluster issuer

Issue: ZENKO-4876

francoisferrand committed:
Test that federated metrics are retrieved

Issue: ZENKO-4876

francoisferrand committed:
Set federated prometheus name

This is needed to avoid conflict with internal prometheus service,
since we are running in the same cluster.

Issue: ZENKO-4876

francoisferrand committed:
Fix prometheus config

ServiceMonitor were not scrapped.

Issue: ZENKO-4876

francoisferrand committed:
Fix zkop install to support sha1

When an SHA1 is used which is not the tip of the branch, it is not
available after a `git clone` (which fetches only commits reachable
from current branches).

This causes issues if a build is started while the component branch with
the commit has been rewritten (rebase...)

To overcome this, the safe way is to explicitely fetch the commit: which
works for any commit (branch, tag, sha1).

Issue: ZENKO-4876

francoisferrand committed:
Merge alerts workflow

No reason to keep separate, and it helps avoid a github bug/limitation
when creating a check: and the

Issue: ZENKO-4876

francoisferrand committed:
Move kafka dump to folder

Issue: ZENKO-4876

francoisferrand committed:
Bump backbeat 8.6.51

Issue: ZENKO-4926

francoisferrand committed:
Bump zenko-operator 1.6.4

Issue: ZENKO-4926

Seragonia committed:
Ensure callback is called once in doWhilst with S3 operations

The trigger seem to be
https://github.com/aws/aws-sdk-js/issues/1678
... causing the callback to be called twice, which
is a problem in these loops.
Issue: ZENKO-4925

Seragonia committed:
Improve variable in doWhilst

Issue: ZENKO-4925

francoisferrand committed:
Bump zenko-operator 1.6.5

Issue: ZENKO-4919

francoisferrand committed:
Bump cloudserver 8.8.36

Issue: ZENKO-4919

francoisferrand committed:
Bump sorbet 1.1.13

Issue: ZENKO-4919

francoisferrand committed:
Merge remote-tracking branch 'origin/bugfix/ZENKO-4925' into improvement/ZENKO-4919

francoisferrand committed:
Merge remote-tracking branch 'origin/improvement/ZENKO-4876' into improvement/ZENKO-4919

francoisferrand committed:
Release Zenko 2.6.70

Issue: ZENKO-4919

bert-e commented:
# Hello francoisferrand,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://bert-e.scality.net/scality/zenko/doc/user).


<details>
  <summary><b>Available options</b></summary>

  name   | description  | privileged   | authored
  ------ | ------------ | ------------ |----------
  `/after_pull_request` | Wait for the given pull request id to be merged before continuing with the current one. |  | 
  `/bypass_author_approval` | Bypass the pull request author's approval |  :star:  | 
  `/bypass_build_status` | Bypass the build and test status |  :star:  | 
  `/bypass_commit_size` | Bypass the check on the size of the changeset ```TBA``` |  :star:  | 
  `/bypass_incompatible_branch` | Bypass the check on the source branch prefix |  :star:  | 
  `/bypass_jira_check` | Bypass the Jira issue check |  :star:  | 
  `/bypass_peer_approval` | Bypass the pull request peers' approval |  :star:  | 
  `/bypass_leader_approval` | Bypass the pull request leaders' approval |  :star:  | 
  `/approve` | Instruct Bert-E that the author has approved the pull request. |  |  :writing_hand: 
  `/create_pull_requests` | Allow the creation of integration pull requests. |  | 
  `/create_integration_branches` | Allow the creation of integration branches. |  | 
  `/no_octopus` | Prevent Wall-E from doing any octopus merge and use multiple consecutive merge instead |  | 
  `/unanimity` | Change review acceptance criteria from `one reviewer at least` to `all reviewers` |  | 
  `/wait` | Instruct Bert-E not to run until further notice. |  | 
  

</details>



<details>
  <summary><b>Available commands</b></summary>

  name   | description  | privileged
  ------ | ------------ | ------------
  `/help` | Print Bert-E's manual in the pull request. | 
  `/status` | Print Bert-E's current status in the pull request ```TBA``` | 
  `/clear` | Remove all comments from Bert-E from the history ```TBA``` | 
  `/retry` | Re-start a fresh build ```TBA``` | 
  `/build` | Re-start a fresh build ```TBA``` | 
  `/force_reset` | Delete integration branches & pull requests, and restart merge process from the beginning. | 
  `/reset` | Try to remove integration branches unless there are commits on them which do not appear on the source branch. | 
  
</details>




*Status report is not available.*

bert-e commented:
# Branches have diverged



This pull request's source branch `improvement/ZENKO-4919` has diverged from
`development/2.10` by more than 50 commits.

To avoid any integration risks, please re-synchronize them using one of the
following solutions:

* Merge `origin/development/2.10` into `improvement/ZENKO-4919`
* Rebase `improvement/ZENKO-4919` onto `origin/development/2.10`

Note: If you choose to rebase, you may have to ask me to rebuild
integration branches using the `reset` command.

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-4919 contains:


* `2.10.6`

* `2.6.69`

* `2.7.65`

* `2.8.45`

* `2.9.23`



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.10.7`

* `2.6.70`

* `2.7.66`

* `2.8.46`

* `2.9.24`


Please check the `Fix Version/s` of ZENKO-4919, or the target
branch of this pull request.

bert-e commented:
# Request integration branches



Waiting for integration branch creation to be requested by the user.

To request integration branches, please comment on this pull request with the following command:

```
/create_integration_branches
```

Alternatively, the `/approve` and `/create_pull_requests` commands will automatically
create the integration branches.

francoisferrand committed:
Fix flaky cluster issuer setup

We need to retry `kubectl apply`, esp as  mutating web hooks may not be
ready, c.f. [1].

[1] https://github.com/cert-manager/cert-manager/issues/2908

Issue: ZENKO-4919

francoisferrand commented:
/approve

bert-e commented:
# History mismatch



Merge commit #f04053760ca79b0a51bad210115c6b65069e7329 on the integration branch
`w/2.7/improvement/ZENKO-4919` is merging a branch which is neither the current
branch `improvement/ZENKO-4919` nor the development branch
`development/2.7`.

It is likely due to a rebase of the branch `improvement/ZENKO-4919` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**





*The following options are set:* **approve**

bert-e commented:
# History mismatch



Merge commit #f04053760ca79b0a51bad210115c6b65069e7329 on the integration branch
`w/2.7/improvement/ZENKO-4919` is merging a branch which is neither the current
branch `improvement/ZENKO-4919` nor the development branch
`development/2.7`.

It is likely due to a rebase of the branch `improvement/ZENKO-4919` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**





*The following options are set:* **approve**

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/Zenko/commit/29f7c323930ba8a8d0a1379b89019c0b8630c457) did not succeed in branch w/2.7/improvement/ZENKO-4919.




*The following options are set:* **approve**

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/Zenko/commit/1add1a4d2be9684ddc7a597229f7b89bbe2c8ccb) did not succeed in branch w/2.8/improvement/ZENKO-4919.




*The following options are set:* **approve**

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/Zenko/commit/29f7c323930ba8a8d0a1379b89019c0b8630c457) did not succeed in branch w/2.7/improvement/ZENKO-4919.




*The following options are set:* **approve**

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/Zenko/commit/c26e9ccc19dfdf2a0931fcd111752bff0e3d15a3) did not succeed in branch w/2.10/improvement/ZENKO-4919.




*The following options are set:* **approve**

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/Zenko/commit/29f7c323930ba8a8d0a1379b89019c0b8630c457) did not succeed in branch w/2.7/improvement/ZENKO-4919.




*The following options are set:* **approve**

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/Zenko/commit/2a31a02ee7cbe8a7dac1053b417d93628c5934a8) did not succeed in branch w/2.9/improvement/ZENKO-4919.




*The following options are set:* **approve**

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/Zenko/commit/29f7c323930ba8a8d0a1379b89019c0b8630c457) did not succeed in branch w/2.7/improvement/ZENKO-4919.




*The following options are set:* **approve**

bert-e commented:
# Build failed



The build for [commit](https://github.com/scality/Zenko/commit/2a31a02ee7cbe8a7dac1053b417d93628c5934a8) did not succeed in branch w/2.9/improvement/ZENKO-4919.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/2.6`

* :heavy_check_mark: `development/2.7`

* :heavy_check_mark: `development/2.8`

* :heavy_check_mark: `development/2.9`

* :heavy_check_mark: `development/2.10`



The following branches have **NOT** changed:

* `development/2.5`



Please check the status of the associated issue ZENKO-4919.

Goodbye francoisferrand.




*The following options are set:* **approve**
